docs/organizer-guide/confs/registration.rst

```diff
@@ -0,0 +1,110 @@
+"temporary attendee". These are for attendees that are allowed
+to enter for a limited time, but do not have a conference ticket, like the
+friend of a speaker coming to see a single talk.
+
+Lanyards
+--------
+
+Write the Docs conferences use different colors for lanyards:
+
+* Black for regular attendees who are ok with being in photos.
+* White for regular attendees who do not want photographs
+  published.
+* Yellow for conference staff.
+* Purple for on duty volunteers (may be worn off duty too).
+* Red for vendors.
+* Green for temporary badges.
+
+Tasks of the on-site registration manager
+-----------------------------------------
+
+The on-site registration manager should be at the registration desk during
+the busiest times to solve any issues, and have access to Tito to look up
+registration details. During the night, they should move all attendee lists
+and badges to a locked room.
+
+.. _conf-registration-volunteers:
+
+Registration process for volunteers
+-----------------------------------
+
+For each attendee, the volunteers should:
+
+* Check whether the person is in the attendee list, and cross
+  them off from the list.
+* Find the attendee's badge and hand it to them.
+* Explain the difference between black and white lanyards, and let
+  the attendee take whichever they prefer. Staff and vendors are marked as such
+  in the attendee list, and they get a yellow or red lanyard.
+* Roughly point out the location of the main conference, cloak room,
+  toilets, badge flair and any swag they can take.
+* If someone is in the wrong last name queue, direct them to the
+  correct one (they don't have to go to the back of the queue).
+* Temporary attendees will be on a separate list, including when they
+  can enter. They get a temporary badge with green lanyard.
+
+Important notes:
+
+* Some attendees will bring a printed ticket. This is not required, but
+  can be helpful in correctly guessing the spelling of their last name.
+* If an attendee is in the attendee list, but does not have a badge,
+  they likely registered too late or changed the registration.
+  They can take a blank badge to write their name on it.
+* If an attendee is not in the attendee list, they can not register,
+] **even if they do have a badge**. An organiser will have to be involved.
+  The exception to this is when they say they are taking someone else's place,
+  and that person is on the list. In that case, cross off that person, write
+  the attendee's name next to it, and let the attendee write their name on a
+  blank badge.
+
+Regardless of the number of signs, people will often not queue up correctly
+by last name, so it may be needed to direct people to the right queue,
+especially during busy moments.
+
+If registration gets particularly busy, the on-site registration manager or
+floater can already hand out white or black lanyards to people in the queue,
+guide them to split up correctly, and provide other information like the venue
+layout.
```

Comment:
💯 document. I wonder if there's an easy way to make this into a cheatsheet or checklist that we can actually print and put on the reg desk for each volunteer. Would be neat to have this for each role, but reg seems simplest to start.